lib/simply_orm.rb

```diff
@@ -0,0 +1,94 @@
+class SimplyORM
```

Comment:
Its very strange that my last commit contains this file related to ORM but locally in git I don't stage this file. As you can see on attachment, my commit doesn't include 'lib' folder
![p70317-211740](https://cloud.githubusercontent.com/assets/20019225/24059236/fdfb0296-0b56-11e7-9362-2708fdc3b70a.jpg)